src/state_machine/signer/mod.rs

```diff
@@ -555,8 +554,11 @@ impl<SignerType: SignerTrait> Signer<SignerType> {
         let response = Message::NonceResponse(response);
 
         info!(
-            "Signer {} sending NonceResponse for DKG round {} sign round {} sign iteration {}",
-            signer_id, nonce_request.dkg_id, nonce_request.sign_id, nonce_request.sign_iter_id,
+            %signer_id,
```

Comment:
This is one of the lines that could use some extra attention.

Reply:
Can you please clarify what your recommendation would be in this specific instance?

Reply:
@xoloki do you mean having these parameters laid out on one line, instead of having each on their own line?

Reply:
This is what this line currently looks like:
```
sending NonceResponse signer_id=0 nonce_request.dkg_id=1 nonce_request.sign_id=1 nonce_request.sign_iter_id=1
```
I don't think we need the `nonce_request.` tags in the log line.  In general, we shouldn't put extraneous info into the logs, so unless we're logging both `self.dkg_id` and `nonce_request.dkg_id` (e.g. to show they differ), we can just log `%dkg_id=nonce_request.dkg_id`.

Reply:
Thanks for the clarification.